.vscode/tasks.json

```diff
@@ -2,16 +2,143 @@
+            "command": "dotnet",
+            "type": "process",
+            "dependsOn": ["Build MonoGame Framework Content Pipeline"],
+            "args": [
+                "build",
+                "${workspaceFolder}/MonoGame.Framework/MonoGame.Framework.ConsoleCheck.csproj",
+                "/property:GenerateFullPaths=true"
+            ],
+            "problemMatcher": "$msCompile"
+        },
+        {
+            "label": "Build MonoGame Framework (DesktopGL)",
+            "command": "dotnet",
+            "type": "process",
+            "dependsOn": ["Build MonoGame Framework Content Pipeline"],
+            "args": [
+                "build",
+                "${workspaceFolder}/MonoGame.Framework/MonoGame.Framework.DesktopGL.csproj",
+                "/property:GenerateFullPaths=true"
+            ],
+            "problemMatcher": "$msCompile"
+        },
+        {
+            "label": "Build MonoGame Framework (IOS)",
+            "command": "dotnet",
+            "type": "process",
+            "dependsOn": ["Build MonoGame Framework Content Pipeline"],
+            "args": [
+                "build",
+                "${workspaceFolder}/MonoGame.Framework/MonoGame.Framework.IOS.csproj",
+                "/property:GenerateFullPaths=true"
+            ],
+            "problemMatcher": "$msCompile"
+        },
+        {
+            "label": "Build MonoGame Framework (WindowsDX)",
+            "command": "dotnet",
+            "type": "process",
+            "dependsOn": ["Build MonoGame Framework Content Pipeline"],
+            "args": [
+                "build",
+                "${workspaceFolder}/MonoGame.Framework/MonoGame.Framework.WindowsDX.csproj",
+                "/property:GenerateFullPaths=true"
+            ],
+            "problemMatcher": "$msCompile"
+        },
+        {
+            "label": "Build MonoGame Framework (WindowsUniversal)",
+            "command": "dotnet",
+            "type": "process",
+            "dependsOn": ["Build MonoGame Framework Content Pipeline"],
+            "args": [
+                "build",
+                "${workspaceFolder}/MonoGame.Framework/MonoGame.Framework.WindowsUniversal.csproj",
+                "/property:GenerateFullPaths=true"
+            ],
+            "problemMatcher": "$msCompile"
+        },
+        {
+            "label": "Build MonoGame Framework Content Pipeline",
+            "command": "dotnet",
+            "type": "process",
+            "args": [
+                "build",
+                "${workspaceFolder}/MonoGame.Framework.Content.Pipeline/MonoGame.Framework.Content.Pipeline.csproj",
+                "/property:GenerateFullPaths=true"
+            ],
+            "problemMatcher": "$msCompile"
+        },
+        {
+            "label": "Build mgcb-editor",
+            "command": "dotnet",
+            "type": "process",
+            "dependsOn": ["Build MonoGame Framework Content Pipeline"],
```

Comment:
Don't add any `dependsOn`, `dotnet build` will already resolve the dependency while building the `.csproj` so now we are just slowing down the build process.